### PR TITLE
Presentation of results

### PR DESCRIPTION
**Pull request by canyon289**
Added business presentation. Still look for a good paper to do "academic" example.

### CODE REVIEW

**Comment by OriolAbril** [comment]
Shouldn't the ppc plot use histograms instead of kde? I thought ArviZ chose depending on the dtype of the data. Am I not remembering correctly or is there a bug somewhere?

**Comment by canyon289** [comment]
Oh yea you're right it should happen here. Seems like there's a bug

https://github.com/arviz-devs/arviz/blob/master/arviz/plots/ppcplot.py#L303-L314

**Comment by canyon289** [comment]
I'll create a minimal example and file issue ticket in ArviZ before merging

File: content/Section_05/Business_Presentation.ipynb [hunk 1]
```diff
@@ -0,0 +1,356 @@
+{
```
Comment (aloctavodia) [inline content/Section_05/Business_Presentation.ipynb]: <p>There is no normality assumption on the computation of the mean or standard deviation. I guess what you want to convey is the idea that the standard deviation can be missleading if the user assumes normality, specially for a skewed distribution, but that's why we have the hpd, that works ok for non symetrical distributions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/Exploratory-Analysis-of-Bayesian-Models/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (aloctavodia) [inline content/Section_05/Business_Presentation.ipynb]: <p>I do not see why the sentence  "The average sales per day will be between 28 and 32 newspaper with a probability of 0.94" is not equivalent to the statement  "94% of days will have newspaper sales between 28 and 32"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/Exploratory-Analysis-of-Bayesian-Models/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (canyon289) [reply content/Section_05/Business_Presentation.ipynb]: <p>Yea thats what I meant. If you only look at az.summary you might miss that the distribution is something non symmetrical like exponential. I'll reword</p>

Comment (ahartikainen) [reply content/Section_05/Business_Presentation.ipynb]: Yeah, that's why we need also more quantiles :)

Comment (canyon289) [reply content/Section_05/Business_Presentation.ipynb]: <p>Thanks for implicity feedback that I need to reword, let me try explaining in this comment to see if it makes sense.</p><p> </p><p>When I show a posterior plot like this sometimes my non statistician colleagues think on 94% of days we will sell between 28 and 32 newspaper, then get surprised when theres many days were we well 20, or 18.</p><p> </p><p>What they dont realize, and I used to do a poor job explaining, is that the posterior is a distribution of parameters, and that if lambda=28, that just means there's poisson distribution with its own uncertainty centered around 28.</p><p> </p><p> </p><p>In other words (distribution of newspapers sold !=distribution of likely lambda). That's particularly evident when you compare the posterior predictive plot to the posterior plot.</p><p> </p><p>In math notation I believe this is p(y'|y) != p(lambda |y)</p>

Comment (canyon289) [reply content/Section_05/Business_Presentation.ipynb]: <p>I will reword this too! Thanks for feedback Osvaldo and Ari!</p>

Comment (aloctavodia) [reply content/Section_05/Business_Presentation.ipynb]: Right!

Comment (aloctavodia) [reply content/Section_05/Business_Presentation.ipynb]: btw, maybe removing the bullet points and writing in a more verbose or explicit way will be easier.

Comment (canyon289) [reply content/Section_05/Business_Presentation.ipynb]: Is it bad right now? Just let me know, I don't mind editing just want to do what's best

Comment (aloctavodia) [reply content/Section_05/Business_Presentation.ipynb]: I think we should try to write it in a more "book-style" and less "slides-style"

Comment (canyon289) [reply content/Section_05/Business_Presentation.ipynb]: I made changes! Let me know if you like the tone. Its casual and refers to both myself and the reader. Not sure if you want it to be more academic though.

Btw this is good practice for when we start CRC book :D

**Comment by canyon289** [comment]
Also will rerun notebook with PyMC3 fix so PPC plot is histogram before merging

File: content/Section_05/Business_Presentation.ipynb (hunk 1 above)
Comment (aloctavodia) [reply content/Section_05/Business_Presentation.ipynb]: I like it!

**Comment by aloctavodia** [comment]
PyMC3 fix is merged. You can rerun the notebook and feel free to merge after that

**Comment by canyon289** [comment]
Thanks Osvlado and Oriol!

**Comment by canyon289** [comment]
PPC plot fixed. Thanks Osvaldo for pymc3 bugfix.
